compiler/qsc_data_structures/src/namespaces/tests.rs

```diff
@@ -329,3 +329,190 @@ fn test_get_namespace_id() {
     "#]]
     .assert_debug_eq(&result_buf);
 }
+
+#[allow(clippy::too_many_lines)]
+#[test]
+fn test_tree_iter() {
+    let mut root = NamespaceTreeRoot::default();
+    for i in 0..3 {
+        for j in 'a'..'d' {
+            let _ = root.insert_or_find_namespace(
```

Comment:
Given the changes in `namespaces.rs`, was this test meant for `insert_or_find_namespace_from_root_with_id`?